index.html

```diff
@@ -7,10 +7,39 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Portfolio Site</title>
     <link rel="stylesheet" href="./style.css">
-    <link rel="icon" href="./favicon.ico" type="image/x-icon">
+    <link href='https://fonts.googleapis.com/css?family=Playfair Display' rel='stylesheet'>
+    <link href="https://fonts.cdnfonts.com/css/poppins" rel="stylesheet">
 </head>
 
 <body>
+    <nav>
+        <div class="menu">
+            <div>
+                <img id="triangle" src="./assets/img/triangle.png" alt="Triangle">
+                <img id="square" src="./assets/img/square.png" alt="square">
+            </div>
+            <div>
+                <img id="menuimg" src="./assets/img/menu.png" alt="square">
+            </div>
+        </div>
+    </nav>
+    <main>
+        <section id="intro">
+            <div>
+                <div class="title">
+                    <h1>Charlie Giorgania</h1>
```

Comment:
- [ ] The style of the main header is nice, but this is your portfolio and you should create and populate it with information about you because it will display your projects and help you get hired in the future. So kindly change the template title and use your name instead.